matchbox_socket/src/webrtc_socket/native/message_loop.rs

```diff
@@ -424,6 +426,7 @@ async fn create_data_channel(
     let config = RTCDataChannelInit {
         ordered: Some(false),
         max_retransmits: Some(0),
+        negotiated: Some(124),
```

Comment:
124 is more obviously a "protocol" hardcoded id, then 0, that may look like a default value. Non-negotiated channels was getting complex very quickly and had to increase the handshake chatter. I would've needed to make a more scary PR. But we can definitely support non-negotiated channels in a next iteration.

Reply:
Non-negotiated is fine. Whatever works and makes the code easy to understand :)

Since it's shared between the two impls: could you make a constant for it in `webrtc_socket/mod.rs`?

Reply:
Good call, I'll add the constant.